### PR TITLE
Refactor basic launch grading related configuration

### PR DESCRIPTION
**Pull request by marcospri**
This just moves around some code in LTI launch to make  all the grading related parts to be together.

### CODE REVIEW

File: lms/views/lti/basic_launch.py [hunk 1]
```diff
@@ -177,6 +177,9 @@ def _show_document(self, document_url, assignment_extra=None):
             document_url, extra=assignment_extra, is_gradable=assignment_gradable
         )
 
+        # Set up grading related config
+        self._configure_grading(document_url, assignment)
```
Comment (marcospri) [inline lms/views/lti/basic_launch.py:181]: All grading related stuff in one of these methods now.

File: lms/views/lti/basic_launch.py [hunk 2]
```diff
@@ -259,6 +262,10 @@ def _configure_js_for_file_picker(
         )
 
     def _configure_js_to_show_document(self, document_url, assignment):
+        self.context.js_config.add_document_url(document_url)
+        self.context.js_config.enable_lti_launch_mode(self.course, assignment)
+
+    def _configure_grading(self, document_url, assignment):
```
Comment (marcospri) [inline lms/views/lti/basic_launch.py:268]: Grading is Canvas vs the rest.

This probably should be "productized" but I reckon it's better to do it in two steps, refactor it so it's all in one place like I'm trying to do here and then move it to the product.

File: lms/views/lti/basic_launch.py [hunk 3]
```diff
@@ -278,29 +285,25 @@ def _configure_js_to_show_document(self, document_url, assignment):
...
                 self.context.js_config.set_focused_user(focused_user)
 
-        elif self.request.lti_user.is_instructor:
-            # nb. Canvas does not currently use/support any functionality from the
-            # instructor toolbar. For grading it uses SpeedGrader and we don't
-            # support editing assignments.
+        else:
+            self._configure_grading_bar(assignment)
+
+    def _configure_grading_bar(self, assignment):
```
Comment (marcospri) [inline lms/views/lti/basic_launch.py:291]: Grading bar, instructor vs learners.

File: lms/views/lti/basic_launch.py (hunk 2 above)
Comment (marcospri) [reply lms/views/lti/basic_launch.py:268]: If we do the "productization" I reckon "_configure_grading" and "_configure_grading_bar" will merge into one method.

**Review by jon-betts — COMMENTED** [review]
I've just noticed I had some pending review comments on this. Pushing this through now. Let me know if you want a proper review.

**Comment by marcospri** [comment]
Replaced by https://github.com/hypothesis/lms/pull/5623

**Pull request closed without merging**